x-pack/plugins/observability_solution/infra/public/components/lens/lens_chart.tsx

```diff
@@ -92,7 +92,10 @@ export const LensChart = React.memo(
                         defaultMessage="To display this chart, please ensure you are collecting the following fields:"
                       />
                     </p>
-                    <p>{missingFieldsMessage && missingFieldsMessage.longMessage}</p>
+                    <p>
```

Comment:
This is worth a follow-up (or feel free to help in this PR) 

The `longMessage` is typed as `React.ReactNode | ((closePopover: () => void) => React.ReactNode);` but used in the most places simply as a `ReactNode`, which is incorrect

Ideally, this should simply be ` ReactNode` or a `typeof === function` case should be handled with every render.